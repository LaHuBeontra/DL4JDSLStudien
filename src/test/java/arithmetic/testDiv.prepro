function main() {
    import vec3 p1, vec3 p2;

    vec3 div = p1 / p2;

    export div;
}

