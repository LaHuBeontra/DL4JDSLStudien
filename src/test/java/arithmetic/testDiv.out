PreProDataSet{
dataSet=[0.98,   NaN,   NaN,  0.00,   NaN,   NaN,  0.83,   NaN,   NaN,  0.86,   NaN,   NaN]
variableNames=[div]
variableShapes=[4, 3]}